src/Widgets/calendar/ControlHeader.vala

```diff
@@ -25,28 +25,50 @@ namespace DateTime.Widgets {
         public signal void center_clicked ();
         public ControlHeader () {
             Object (orientation : Gtk.Orientation.HORIZONTAL);
+
+            var label = new Gtk.Label (new GLib.DateTime.now_local ().format (_("%OB, %Y")));
+            label.get_style_context ().add_class (Granite.STYLE_CLASS_H2_LABEL);
+            label.halign = Gtk.Align.START;
+            label.set_size_request (185, -1);
+            label.xalign = 0;
+
             var left_button = new Gtk.Button.from_icon_name ("pan-start-symbolic");
+            var center_button = new Gtk.Button.from_icon_name ("office-calendar-symbolic");
+            center_button.tooltip_text = _("Go to today's date");
             var right_button = new Gtk.Button.from_icon_name ("pan-end-symbolic");
-            var center_button = new Gtk.Button.with_label (new GLib.DateTime.now_local ().format (_("%OB %Y")));
+
+            var box_buttons = new Gtk.Grid ();
+            box_buttons.halign = Gtk.Align.END;
+            box_buttons.valign = Gtk.Align.CENTER;
+            box_buttons.add (left_button);
+            box_buttons.add (center_button);
+            box_buttons.add (right_button);
+
             CalendarModel.get_default ().parameters_changed.connect (() => {
                 var date = CalendarModel.get_default ().month_start;
-                center_button.set_label (date.format (_("%OB %Y")));
+                label.set_label (date.format (_("%OB, %Y")));
             });
+
+            var grid = new Gtk.Grid ();
+            grid.orientation = Gtk.Orientation.VERTICAL;
+            grid.column_spacing = 24;
+            grid.margin = 6;
+            grid.attach (label, 0, 0, 1, 1);
```

Comment:
If you're just packing these next to each other horizontally, why not just use `grid.add ()`?

Reply:
I can't change individual items's margins if I use `add ()`. It's dumb.